### PR TITLE
chore: remove references to "malleated"

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/celestia-app/issues/1253

### CODE REVIEW

File: x/blob/README.md [hunk 1]
```diff
@@ -23,10 +23,6 @@ When a `MsgPayForBlob` is processed, it consumes gas based on the blob size.
 
 - [`MsgPayForBlob`](https://github.com/celestiaorg/celestia-app/blob/8b9c4c9d13fe0ccb6ea936cc26dee3f52b6f6129/proto/blob/tx.proto#L39-L44) pays for the blob to be included in the block.
 
-## PrepareProposal
-
-The malleation process occurs during the PrepareProposal step.
```
Comment (cmwaters) [inline x/blob/README.md:28]: I would still have a section here talking about how the PFB gets put in a wrapper that also contains the index(es) of the blob(s) that it references

Comment (rootulp) [reply x/blob/README.md:28]: proposal to tackle this in a stand-alone PR b/c there are several things in this file that need updating and they're not all related to malleation: https://github.com/celestiaorg/celestia-app/issues/1268

File: app/encoding/index_wrapper_decoder.go [hunk 2]
```diff
@@ -5,10 +5,10 @@ import (
 	coretypes "github.com/tendermint/tendermint/types"
 )
 
-func WrappedTxDecoder(dec sdk.TxDecoder) sdk.TxDecoder {
+func IndexWrapperDecoder(dec sdk.TxDecoder) sdk.TxDecoder {
```
Comment (evan-forbes) [inline app/encoding/index_wrapper_decoder.go:8]: [super minor no blocking comment]
I know no one uses this function, but does this mean this is technically API breaking? (so chore! ?)

Comment (rootulp) [reply app/encoding/index_wrapper_decoder.go:8]: oof yea it's technically exported public API so this should've been `chore!`

**Pull request merged**